business-wiki-frontend/pages/home/index.tsx

```diff
@@ -0,0 +1,91 @@
+import defaultStyles from "../../shared/styles/Default.module.scss";
+import buttonStyles from "../../shared/styles/Button.module.scss";
+import moduleStyles from "../../shared/styles/Home.module.scss";
+
+import Head from "next/head";
+import Link from "next/link";
+import Footer from "../../shared/components/Footer";
+import React from "react";
+import { UserContext } from "../../shared/contexts/UserContext";
+import { SidepanelListElementInterface } from "../../shared/models/SidepanelListElementInterface";
+import { Page } from "../../shared/models/Page";
+import SidepanelListElement from "../../shared/components/SidepanelListElement";
+import SidepanelListAddElement from "../../shared/components/SidepanelListAddElement";
+import PageForm from "../../shared/components/PageForm";
+import { UserInterface } from "../../shared/models/UserInterface";
+import PageDetails from "../../shared/components/PageDetails";
+
+export default function Home() {
+  const context = React.useContext(UserContext);
```

Comment:
Możesz też zdestrukturyzować context const {user} = useContext(UserContext). Dodatkowo tak samo zdrestrukturyzuj hooka useContext i dodaj go do importu Reacta

Reply:
Resolved in 08f5fe1d74642db7afa6f2aa0d11f943936ec5ed